README.md

```diff
@@ -10,3 +10,18 @@ There are many repositories that are needed to build the Test Delivery System an
 It's really as simple as that.
 
 The clone script will create a new `repositories` directory and clone each repository into there.
+
+## Running TDS Microservices Using Docker Compose
+The docker-compose file located in the /docker/ directory can used to easily create, start, and manage microservices docker containers. 
+
```

Comment:
Can you mention that the values will need to be added to the tds-docker.env and give some sample values maybe.